CorsixTH/Src/th_sound.cpp

```diff
@@ -150,179 +150,179 @@ size_t THSoundArchive::getSoundDuration(size_t iIndex)
 
 #undef FOURCC
 
-SDL_RWops* THSoundArchive::loadSound(size_t iIndex)
+SDL_RWops* th_sound_archive::load_sound(size_t iIndex)
 {
-    if(iIndex >= m_iFileCount)
+    if(iIndex >= sound_file_count)
         return nullptr;
 
-    th_fileinfo_t *pFile = m_pFiles + iIndex;
-    return SDL_RWFromConstMem(m_pData + pFile->position, pFile->length);
+    th_fileinfo *pFile = sound_files + iIndex;
```

Comment:
th prefix looks wrong/inconsistent

Reply:
renamed completely, file here is misleading at best.